src/query/interpret/eval.hpp

```diff
@@ -837,6 +838,7 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
 
   TypedValue Visit(Function &function) override {
```

Comment:
Do you know whether Visit will enter here only for mgp_func or something else also

Reply:
Built-in functions too